modules/creating-a-machine-pool-cli.adoc

```diff
@@ -18,6 +18,7 @@ You can create additional machine pools for your {product-title} (ROSA) cluster
 
 * To add a machine pool that does not use autoscaling, create the machine pool and define the instance type, compute (also known as worker) node count, and node labels:
 +
+--
```

Comment:
Noting for the future: _[Open blocks](https://docs.asciidoctor.org/asciidoc/latest/blocks/open-blocks)_ (the shave been wrapped around both of the code blocks in this procedure, so that the callouts behave as expected (namely, allows for `+` multi-paragraph continuations and admonition blocks, without breaking callout number order or indentation).